src/cluster_legacy.c

```diff
@@ -1000,7 +1061,12 @@ int clusterLockConfig(char *filename) {
 }
 
 /* Derives our ports to be announced in the cluster bus. */
-void deriveAnnouncedPorts(int *announced_tcp_port, int *announced_tls_port, int *announced_cport) {
+void deriveAnnouncedPorts(
+    int *announced_tcp_port,
+    int *announced_tls_port,
+    int *announced_cport,
+    int *announced_client_tcp_port,
+    int *announced_client_tls_port) {
```

Comment:
Clang-format accepts this too, I think:

```suggestion
void deriveAnnouncedPorts(int *announced_tcp_port,
                          int *announced_tls_port,
                          int *announced_cport,
                          int *announced_client_tcp_port,
                          int *announced_client_tls_port) {
```

I prefer it...